modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItem.js

```diff
@@ -169,97 +169,49 @@ function CartItem({
 
 	const {isGettingRemoved, isRemovalCanceled, isRemoved} = itemState;
 
-	const getClassName = (className) => {
-		return classnames(className, {
-			'mini-cart-item-alignment':
-				Liferay.FeatureFlags['COMMERCE-9599'] ||
-				Liferay.FeatureFlags['COMMERCE-11287'],
-		});
-	};
-
 	return (
 		<div
 			className={classnames('mini-cart-item', {
-				'align-items-start':
-					Liferay.FeatureFlags['COMMERCE-9599'] && hasChildItems,
+				'align-items-start': hasChildItems,
 				'is-removed': isRemoved,
 			})}
 		>
-			{Liferay.FeatureFlags['COMMERCE-9599'] ||
```

Comment:
Based on last chat with @riccardo-alberti I think we need to keep the "new layout" using the feature flag. @andrea-ale-sbarra what do you think?

Reply:
But if I remember correctly the accordion you are reusing is currently under a feature flag (the same of the "edit option" and you have to enable it to see the new layout, otherwise you see the "old view"). With this PR you're removing this control and the accordion will always be used.

You can absolutely combine your changes with the existing accordion, but as far as I know the accordion should remain under the feature flag. But the final decision is certainly up to @andrea-ale-sbarra  😄

Reply:
yes this fix is under COMMERCE-9599 feature flag